### PR TITLE
chore: code style 2nd part

### LINKED ISSUE #26: chore: code style
If this doesn’t look right, [choose a different type.](https://github.com/JDRaftKeeper/RaftKeeper/issues/new)

### Description
 code style fix

### PR DESCRIPTION
**Pull request by JackyWoo**
### Which issues of this PR fixes ：

This PR try to fix #26 

### Change log

1. fix some TODO
2. add some log
3. othe issues about code style

### CODE REVIEW

File: src/Service/NuRaftLogSegment.cpp [hunk 1]
```diff
@@ -1056,16 +1015,18 @@ int LogSegmentStore::removeSegment(UInt64 first_index_kept)
             }
         }
 
-        for (size_t i = 0; i < remove_vec.size(); ++i)
+        for (auto & i : remove_vec)
```
Comment (nicelulu) [inline src/Service/NuRaftLogSegment.cpp:1018]: rename i

File: src/Service/KeeperStore.cpp [hunk 2]
```diff
@@ -995,8 +971,8 @@ struct SvsKeeperStorageGetACLRequest final : public StoreRequest
 
     using StoreRequest::StoreRequest;
 
-    std::pair<Coordination::ZooKeeperResponsePtr, Undo> process(KeeperStore & store, int64_t /*zxid*/, int64_t /*session_id*/,
-                                                                int64_t /* time */) const override
+    std::pair<Coordination::ZooKeeperResponsePtr, Undo>
+    process(KeeperStore & store, int64_t /*zxid*/, int64_t /*session_id*/, int64_t /* time */) const override
```
[the remark this answers is not in the document]
Comment (JackyWoo) [reply src/Service/KeeperStore.cpp:975]: it is fine

File: tests/integration/test_nodes_remove/test.py [hunk 3]
```diff
@@ -91,6 +93,9 @@ def test_nodes_remove(started_cluster):
     node2.copy_file_to_container(os.path.join(CONFIG_DIR, "enable_single_keeper1.xml"),
                                  "/etc/raftkeeper-server/config.d/enable_keeper2.xml")
 
+    # sleep at least 3s, for ConfigReloader monitor config file change every 2s
+    time.sleep(3)
```
Comment (nicelulu) [inline tests/integration/test_nodes_remove/test.py:97]: Don't need sleep

File: cmake/linux/toolchain-x86_64.cmake [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+set (CMAKE_SYSTEM_NAME "Linux")
+set (CMAKE_SYSTEM_PROCESSOR "x86_64")
+set (CMAKE_C_COMPILER_TARGET "x86_64-linux-gnu")
+set (CMAKE_CXX_COMPILER_TARGET "x86_64-linux-gnu")
+set (CMAKE_ASM_COMPILER_TARGET "x86_64-linux-gnu")
+
+# Will be changed later, but somehow needed to be set here.
+set (CMAKE_AR "ar")
+set (CMAKE_RANLIB "ranlib")
```
Comment (nicelulu) [inline cmake/linux/toolchain-x86_64.cmake:24]: dupliated with https://github.com/JDRaftKeeper/RaftKeeper/pull/48

**Comment by nicelulu** [comment]
please resolve conflict

**Review by nicelulu — APPROVED** [review]

**Pull request merged**